pandas/tests/arrays/integer/test_arithmetic.py

```diff
@@ -233,6 +233,13 @@ def test_error_invalid_values(data, all_arithmetic_operators):
 # TODO test unsigned overflow
 
 
+def test_rmod_large_series():
+    result = pd.Series([2] * 10001).rmod(-1)
```

Comment:
Could you add the GH reference, e.g. first line of the test

# GH#issue number